lambdatest/README.md

```diff
@@ -25,13 +25,17 @@ All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datad
 
 Here's how you can track incidents in Datadog with LambdaTest:
 
-1. Log in to your LambdaTest account on the LambdaTest website.
-2. Select settings from the menu bar.
-3. Click on integrations under settings.
-4. To integrate Datadog, click on install next to Push to Datadog.
-5. Enter your [Datadog API and application key][2].
-6. A confirmation email is sent. Confirm the integration in the email.
-7. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
+1. Click **Connect Accounts** to begin authorization of the LambdaTest integration from the Login page in LambdaTest.
+2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to the Datadog authorization page.
```

Comment:
```suggestion
2. Log in to your LambdaTest account on the LambdaTest website to be redirected to the Datadog authorization page.
```

Reply:
Please remove the future tense here as mentioned in our style guide:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense